src/main/java/F/Main.java

```diff
@@ -0,0 +1,9 @@
+package F;
+
+import ua.lviv.iot.lab2.manager.RenovationCompany;
+
+public class Main {
+    public static void main(String[] args) {
+        RenovationCompany renovationCompany_one = new RenovationCompany();
```

Comment:
I'm ok with an underscore inside variable/object/method names as long as you keep your code style consistent. In your ManagerTest you do things like `cable1`. So, consider renaming `renovationCompany_one` to `renovationCompany1` or changing all the other variables.